src/PlatformBar.js

```diff
@@ -11,14 +12,24 @@ export default class ConsoleBar extends Component {
     let checked = [];
     const checkboxes = document.querySelectorAll('.platformCheckBox:checked')
     checkboxes.forEach((elem) => {
-        checked.push(elem.value)
+      checked.push(elem.value)
     })
-    this.setState({platformsToFilter: checked});
-    this.props.updateState({platformsToFilter: this.state.platformsToFilter})
-    }
+    this.setState({ platformsToFilter: checked });
+    this.props.updateState({ platformsToFilter: this.state.platformsToFilter })
+  }
   render() {
-    return(
+    return (
       <div className="console-bar search-section">
+      <div className="platform-check">
+        {
+          this.props.platforms.map((val, i) => {
```

Comment:
PlatformBar.js line 25, I really love the idea of iterating through our platforms and appending an html element for each rather than just manually creating them all ourselves. Great logic!

Reply:
Plus it will allow us to add my games to our dataset and adapt to any consoles we don't have on our list